duckduckgo/iOS
# Update to Xcode 10 and Swift 4.2

Task/Issue URL: https://app.asana.com/0/361428290920652/733395012492692

**Description**:
Update codebase to support XCode 10 and Swift 4.2. Once this PR is merged I'll do an internal / Beta release to TestFlight.

**Steps to test this PR**:
1. Test everything! Run the app across devices and check that everything still works as it did before.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
